src/main/java/com/exception/BookPlaceManager.java

```diff
@@ -0,0 +1,11 @@
+package com.exception;
+
+public class BookPlaceManager {
+    public Place book(int placeNumber) throws IncorrectPlaceNumberException {
+        if (placeNumber < 20) {
```

Comment:
я б імя методу трішки поправив, щоб воно саме за себе говорило 
```suggestion
    public Place bookPlace(int placeNumber) throws IncorrectPlaceNumberException {
        if (placeNumber < 20) {
```